vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -69,48 +57,42 @@ plotly express standard.
 
     [FirstPage1]: ../../assets/tutorials/dashboard/dashboard21.png
 
-Let's give it a try and see your dashboard in action! Either paste the above code into a Jupyter notebook cell and evaluate it, or create a new Python script called `app.py` and copy the code from
-above into the script. Next, navigate to the directory where the `app.py` file is located using your terminal or command
-prompt. Run the script by executing the command:
+!!! tip "To see the dashboard in action"
 
-```python
-python app.py
-```
+    Paste the above code into a Notebook cell, run the Notebook, and evaluate it.
+
+    ??? tip "Prefer Python scripts to Notebooks?"
```

Comment:
Do we want these kind of nested admonitions? I've never seen them but maybe best to just use one admonition here then?